docker/postlaunchconfig_update.sh

```diff
@@ -117,7 +117,13 @@ fi
 export KEYSTONE_HOST="127.0.0.1:5001"
 
 echo "[ postlaunchconfig_update - Start UWSGI process ] "
-/usr/bin/keystone-all &
+#/usr/bin/keystone-all &
```

Comment:
Leftover? Should be removed?

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/200/commits/98478cb0152167587cbd9dc8df8fb5df2c391179